cwms-data-api/src/main/java/cwms/cda/data/dao/AuthDao.java

```diff
@@ -206,6 +208,49 @@ private String checkKey(String key) throws CwmsAuthException {
         }
     }
 
+    /**
+     *
+     * @param edipi
+     * @return
+     * @throws CwmsAuthException
+     */
+    private String userForEdipi(Long edipi) throws CwmsAuthException {
+        try {
+            return dsl.connectionResult(c-> {
+                setSessionForAuthCheck(c);
+                try (PreparedStatement userForEdipi = c.prepareStatement(USER_FOR_EDIPI)) {
+                    userForEdipi.setLong(1, edipi);
```

Comment:
Hard to follow the long boxing/unboxing but this is where a null check should occur, though the Long.parseLong earlier returns a primitive....

Reply:
Oh, that should just be a primitive anyways. Oops.